### PR TITLE
Update Azure Section per #4177

### LINKED ISSUE #4177: Update "Code First Migrations and Deployment with the Entity Framework in an ASP.NET MVC Application" to current Azure
https://docs.microsoft.com/en-us/aspnet/mvc/overview/getting-started/getting-started-with-ef-using-mvc/migrations-and-deployment-with-the-entity-framework-in-an-asp-net-mvc-application

This article refers to "Windows Azure" in screenshots and alt text. The Azure screenshots are old and no longer apply. There is at least one comment on this article where the user is confused.

### PR DESCRIPTION
**Pull request by isaac2004**
Fixes #4177 - Updated Azure Deployment Section of MVC/EF Tutorial

- Removed Unneeded Images  
- Added New Screenshots for VS 2015 and Azure Portal  
- Cleaned up External Urls where possible  
- Updated Internal Links to use xref

[Internal Review Page](https://review.docs.microsoft.com/en-us/aspnet/mvc/overview/getting-started/getting-started-with-ef-using-mvc/migrations-and-deployment-with-the-entity-framework-in-an-asp-net-mvc-application?branch=pr-en-us-4276)

### CODE REVIEW

**Comment by isaac2004** [comment]
@terrimorton @scottaddie

File: aspnet/mvc/overview/getting-started/getting-started-with-ef-using-mvc/migrations-and-deployment-with-the-entity-framework-in-an-asp-net-mvc-application.md [hunk 1]
```diff
@@ -129,148 +129,120 @@ So far the application has been running locally in IIS Express on your developme
...
 
-You can find the deployed version of the Web.config file on your own computer in *ContosoUniversity\obj\Release\Package\PackageTmp\Web.config*. You can access the deployed *Web.config* file itself by using FTP. For instructions, see [ASP.NET Web Deployment using Visual Studio: Deploying a Code Update](../../../../web-forms/overview/deployment/visual-studio-web-deployment/deploying-a-code-update.md). Follow the instructions that start with "To use an FTP tool, you need three things: the FTP URL, the user name, and the password."
+You can find the deployed version of the Web.config file on your own computer in *ContosoUniversity\obj\Release\Package\PackageTmp\Web.config*. You can access the deployed *Web.config* file itself by using FTP. For instructions, see [ASP.NET Web Deployment using Visual Studio: Deploying a Code Update](xref:web-forms/overview/deployment/visual-studio-web-deployment/deploying-a-code-update.md). Follow the instructions that start with "To use an FTP tool, you need three things: the FTP URL, the user name, and the password."
```
Comment (scottaddie) [inline aspnet/mvc/overview/getting-started/getting-started-with-ef-using-mvc/migrations-and-deployment-with-the-entity-framework-in-an-asp-net-mvc-application.md]: Remove the ".md" file extension from the "xref" link.

File: aspnet/mvc/overview/getting-started/getting-started-with-ef-using-mvc/migrations-and-deployment-with-the-entity-framework-in-an-asp-net-mvc-application.md [hunk 2]
```diff
@@ -280,7 +252,7 @@ In this tutorial you've seen how to enable migrations and deploy the application
 
 Please leave feedback on how you liked this tutorial and what we could improve. You can also request new topics at [Show Me How With Code](http://aspnet.uservoice.com/forums/228522-show-me-how-with-code).
 
-Links to other Entity Framework resources can be found in [ASP.NET Data Access - Recommended Resources](../../../../whitepapers/aspnet-data-access-content-map.md).
+Links to other Entity Framework resources can be found in [ASP.NET Data Access - Recommended Resources](xref:whitepapers/aspnet-data-access-content-map.md).
```
Comment (scottaddie) [inline aspnet/mvc/overview/getting-started/getting-started-with-ef-using-mvc/migrations-and-deployment-with-the-entity-framework-in-an-asp-net-mvc-application.md]: Remove the ".md" file extension from the "xref" link.

**Comment by isaac2004** [comment]
@scottaddie done, sorry for the delay

**Comment by scottaddie** [comment]
@isaac2004 The screenshot for the following step appears to be outdated as well: **Click the SQL Database, and choose Create New or select an existing database**. Can you please update that one too?

**Comment by scottaddie** [comment]
@isaac2004 Thank you for your help with fixing this issue!

**Pull request merged**